.eslintrc

```diff
@@ -23,6 +23,18 @@
         "argsIgnorePattern": "^_",
         "varsIgnorePattern": "^_"
       }
-    ]
+    ],
+    "import/no-unresolved": "error"
+  },
+  "settings": {
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts"]
+    },
+    "import/resolver": {
+      "typescript": {
+        "alwaysTryTypes": true, // always try to resolve types under `<root>@types` directory even it doesn't contain any source code, like `@types/unist`
+        "project": ["tsconfig.prune.json", "packages/*/tsconfig.json"]
```

Comment:
Why do we need tsconfig.prune here?
The entire codebase is covered by the per-project tsconfigs

Reply:
https://github.com/temporalio/sdk-typescript/blob/32c12fd2aaec1acf6866cfe0b47c3893c428cc15/tsconfig.prune.json#L8-L19

These files are failing without this.

Reply:
Can you elaborate? Failing how?

Reply:
https://github.com/temporalio/sdk-typescript/runs/7462992242?check_suite_focus=true#step:6:12

Reply:
Isn’t that due to not building before running eslint?

Reply:
> Isn’t that due to not building before running eslint?

It could be, and it could be better to not requiring building before lint.

Reply:
We’d still need to build afaict because we reference some js files in lib

Reply:
OK then.

Reply:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/251288/180375259-ded4471e-c8ea-49de-b0bd-7443c9c9e858.png">

This didn't change linting errors. I'll add a build step

Reply:
So it means `prune` can be removed here?

Reply:
Removed, thanks